tests/TestData/SmokeTests.LegacyMSTestVB/My Project/AssemblyInfo.vb

```diff
@@ -0,0 +1,18 @@
+﻿Imports System
```

Comment:
Only the vbproj files should need committed, right? I think you can remove most of the test project and leave just the project files and still have the test work.

Reply:
That feedback makes sense. I decided to follow the established pattern which seemed to include the full project.

I'm referencing these projects:
* SmokeTests.MStestCoreBaseline
* SmokeTests.WpfNet5Baseline